osf_tests/factories.py

```diff
@@ -815,3 +815,18 @@ class Meta:
     creator = factory.SubFactory(AuthUserFactory)
 
     is_deleted = False
+
+class ScheduledBannerFactory(DjangoModelFactory):
+    name = factory.Faker('name')
+    default_text = factory.Faker('text')
+    mobile_text = factory.Faker('text')
+    default_photo = factory.Faker('file_name')
+    mobile_photo = factory.Faker('file_name')
+    license = factory.Faker('name')
+    color = 'white'
+    start_date = timezone.now()
+    end_date = start_date
+    #TODO: Make setting dates smarter
```

Comment:
This should work that way in general because start_date starts at midnight and end_date ends at 11:59 if they are the same it should be a default 1 day banner.